src/components/Filter/AgeFilter.tsx

```diff
@@ -0,0 +1,45 @@
+import { FC } from 'react';
+import classNames from 'classnames';
+import { useSearchParams } from 'react-router-dom';
+import { SearchLink } from '../SearchLink';
+import { centuriesFilters } from '../../utils/data';
+
+export const AgeFilter: FC = () => {
+  const [searchParams] = useSearchParams();
+  const centuries = searchParams.getAll('centuries');
+
+  return (
+    <div className="panel-block">
+      <div className="level is-flex-grow-1 is-mobile" data-cy="CenturyFilter">
+        <div className="level-left">
+          {centuriesFilters.map(({ title, params }) => (
+            <SearchLink
+              key={title}
+              title={title}
+              params={{
+                centuries: centuries.includes(title)
+                  ? centuries.filter(item => item !== title)
+                  : [...params.centuries, ...centuries],
```

Comment:
It's difficult to read
Create `const` value for this condition value, and use it here